pkg/crc/network/proxy.go

```diff
@@ -118,16 +129,16 @@ func (p *ProxyConfig) ApplyToEnvironment() {
 	}
 
 	if p.HTTPProxy != "" {
-		os.Setenv("HTTP_PROXY", p.HTTPProxy)
-		os.Setenv("http_proxy", p.HTTPProxy)
+		setEnv("HTTP_PROXY", p.HTTPProxy)
+		setEnv("http_proxy", p.HTTPProxy)
```

Comment:
Maybe go for `setProxyEnv("http_proxy", p.HTTPProxy)` which will set both upper and lowercase variables?

Reply:
done :+1: